aws/ec2/ec2-how-to/find-ubuntu-images.rst

```diff
@@ -0,0 +1,122 @@
+* ARCH: `amd64` or `arm64`
+* VIRT_TYPE: `pv` or `hvm`
+* VOL_TYPE: `ebs-gp2`, `ebs-io1`, `ebs-standard`, or `instance-store`
+
+The given serial for an image (e.g., `20210222`) is also uploaded in place of `current`:
+
+.. code-block::
+   
+   ubuntu/$PRODUCT/$RELEASE/stable/$SERIAL/$ARCH/$VIRT_TYPE/$VOL_TYPE/ami-id
+
+
+Find EKS images with SSM
+------------------------
+Amazon’s Elastic Kubernetes Service (EKS) is a managed Kubernetes service provided
+by AWS that let users run Kubernetes applications in the cloud or on-premises.
+
+Canonical provides minimized Ubuntu images customised for use with EKS
+. These are fully tested release images that cover all Kubernetes versions
+supported by the EKS service.
+
+The latest EKS AMI ID can be found in the SSM parameter store:
+
+.. code-block::
+
+   aws ssm get-parameters --names /aws/service/canonical/ubuntu/eks/20.04/1.26/stable/current/amd64/hvm/ebs-gp2/ami-id
+
+The path follows this format:
+
+.. code-block::
+
+   ubuntu/eks/$RELEASE/$K8S_VERSION/stable/current/$ARCH/$VIRT_TYPE/$VOL_TYPE/ami-id
+
+Ownership verification
+----------------------
+Users can verify that an AMI was published by Canonical by ensuring
+the `OwnerId` field of an image is set to the expected value. Those values are:
+
+* `099720109477` (in the default partition)
+* `513442679011` (in the GovCloud partition)
+* `837727238323` (in the China partition)
+
+This ID is stored in SSM and is discoverable by running:
+
+.. code-block::
+
+   aws ssm get-parameters --names /aws/service/canonical/meta/publisher-id
+
+
+With the value returned by that command, users can then run the `describe-images`
+command against an AMI ID and verify the `OwnerId` field matches the above ID:
+
+.. code-block::
+
+   aws ec2 describe-images --image-ids $AMI_ID
+
+Note that listings on the AWS Marketplace will always show the `OwnerId` as
+Amazon (e.g. `679593333241`). In these cases, users can verify the Amazon ID
+and look for `aws-marketplace/ubuntu` in the `ImageLocation` field.
+
+
+AWS Marketplace
+---------------
+AWS Marketplace is a digital catalogue with thousands of software listings
+from independent software vendors that make it easy to find, test, buy, and
+deploy software that runs on AWS.
+Canonical maintains `image listings <https://aws.amazon.com/marketplace/seller-profile?id=565feec9-3d43-413e-9760-c651546613f2>`_ for recent Ubuntu releases and special flavors (e.g. `Anbox`, `Pro`, `EKS`) on AWS Marketplace.
+
+Customers can also use the AWS Marketplace to launch and subscribe to official
+Ubuntu Pro images that allow users to pay for additional support.
+These Ubuntu Pro images can also be found in the SSM parameter store.
+
+
+The Marketplace images can be found in the SSM parameter store:
+
+.. code-block::
+
+   aws ssm get-parameter --name /aws/service/marketplace/$IDENTIFIER/latest
+
+* IDENTIFIER one of the identifiers (starting with `prod-`)
+
```

Comment:
Change to:
IDENTIFIER: use one of the following identifiers (starting with `prod-`)